src/_data/how_to/get_connected.yml

```diff
@@ -72,6 +72,7 @@ toc:
         paragraphs:
           - text: The <a href="https://drive.google.com/file/d/1STiB4FsIZPTR97use5s9kdZAtWwdjE4l/view" target="_blank">purchase timeline</a> is anticipated to take anywhere from 10 to 14 weeks, assuming that the start date begins with the Official Notice of Receipt and concludes with SIM delivery.
           - text: The installation timeline will vary based on if the agency or subcontractor will be responsible for installing the SIMs.
+          - text: More information and the self-service instructions are available on the <a href="https://resources.calitp.org/mobility-marketplace/FirstNet-Self-Service-Instructions.pdf" target="_blank">Data Plan Procurement Self-Service Instructions</a>.
```

Comment:
@angela-tran @thekaveman @mrose914  I linked the doc to the _title_ of the doc b/c we should try to avoid linking to "here"... which I see the rest of this doc does.

Reply:
@machikoyasuda great catch. Not using "here" for link text should be a part of content guidelines for folks @mrose914, doing so is bad practice and makes the accessibility experience much worse.